RobotNavigation/Maze.cs

```diff
@@ -20,10 +20,20 @@ public enum Cell
        
         private Cell[,] _maze;
 
+        public Maze()
+        {
+
+        }
+
         public Maze(int height, int width)
         {
             Width = width;
             Height = height;
+            drawMaze();
```

Comment:
drawMaze is an odd name.  You're not really drawing it, just intialising the array.  Init() is a better option I think.

Also - what happens if the Width or Length properties are set after the maze has been created?  Does it re-create the array by calling drawMaze()?

Reply:
Yeah when drawMaze() is called it re-creates the Array. If width and height are changed, drawMaze() needs to be called again for anything to change